deploy/runtools/user_topology.py

```diff
@@ -1,14 +1,27 @@
 """ Define your additional topologies here. The FireSimTopology class inherits
 from UserToplogies and thus can instantiate your topology. """
 
-from runtools.firesim_topology_elements import *
```

Comment:
Having `import *` statements in the code caused things like the `rootLogger` global variable to be imported from other modules (and thus making mypy unable to infer it's type). Across the codebase I removed this and explicitly imported each object/class.